src/content/docs/experiment/onmonetization.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: onmonetization event handler
+---
+
+import { LinkOut, Hidden } from "@interledger/docs-design-system";
+import { Badge } from '@astrojs/starlight/components';
+import Specification from '/src/components/docs/Specification.astro'
+import BrowserCompat from '/src/components/docs/BrowserCompat.astro'
+import data from '/src/data/browser-compat-data/onmonetization.json'
+
+<Badge text="HTML DOM API" style="position: relative; top:-30px;" />
+
+By creating an `onmonetization` event handler, your web page can programmatically respond to `monetization` events.
+
+## `window.onmonetization` 
+
+The `onmonetization` event handler is used by the [Web Monetization agent](/resources/glossary/#web-monetization-agent) to process `monetization` events sent to the browser `window` (the event target). 
+
+The `onmonetization` property can be applied to any element and is used by the browser to notify that some `<link>` element is [monetized](/experiment/link-element). 
+
+In a tabbed browser, each tab is represented by its own `window` object.
+
+## Syntax
+
+```js 
+window.onmonetization=functionRef
+```
+
+Where `functionRef` is the event handler function to execute when a `monetization` event fires.
+
+## Example 
+
+Show a message to site visitors to let them know that your page supports Web Monetization. 
+
+```js
+<script type="text/javascript">
+window.onmonetization = function () {
+  alert('This page supports Web Monetization.')
+  }
```

Comment:
And this will get called on each `monetization` event, so we should not recommend this event as this use case.

Recommended way to detect WM support is following:
```ts
const supportsMonetization: boolean = document.createElement('link').relList.supports('monetization');
```

Reply:
...it's part of "DOM API"

Reply:
I'm aligned with what Sid is mentioning. I think the example should state something around these lines:

`Show a message to site visitors to thank them for their support`.